### ISSUE TITLE
Add option to clean the dist path

### ISSUE THREAD

**Issue by w33ble** [egoist/bili#83]
It would be really helpful if you could pass a flag to bili and have it clean out the dist path before generating the output.

This is actually the only reason I'd like to see #82, so if bili did this step, I wouldn't need to wrap it at all. 😺

Happy to take a crack at a PR too. My plan would be to reach for `rimraf`and call it on the output path based on a flag. Open to better ideas.

**Comment by egoist** [comment]
I thought about this flag before but it didn't end up having this because:

- In most cases you don't need to clean anything since files in `dist` will be overriden anyways
- In some cases you may run two bili commands like `bili a.js && bili b.js` in series so you don't want to remove the `dist` folder after the fist one finished.

**Comment by w33ble** [comment]
I agree that in most cases, it's fine not to clean the output path first. Here's my list of justifications:

- Sometimes you change the build, and having to remember to clean up the dist when you do is kind of a gotcha, you risk publishing stray files to npm when you forget. I'm lazy and forgetful, so I like to play it safe and just reset things before the build.
- As for multiple builds, `bili a.js --clean && bili b.js` would give the desired affect. `rm -rf`, or adding another module, is also an option, but having to do that in every package you use bili is annoying. In my case, I have a monorepo, where each package has a build step, and I'd prefer to use 1 command instead of 2 (ie. `bili --clean --outDir whatever ` instead of `rimraf whatever && bili --outDir whatever`). 
- There's also the advantage in that I don't have to keep any custom output directory in sync with any cleanup command ("whatever" in the example above), so there's less room for error.
